src/App.css

```diff
@@ -18,9 +18,37 @@
 .eachRocket {
   display: flex;
   gap: 5px;
+  align-items: center;
 }
 
 .rocket-image {
-  height: 6rem;
-  width: 250px;
+  height: 150px;
+  width: 200px;
+}
+
+.rev-word {
+  background-color: green;
+  color: #fff;
+}
+
+.rocket-info {
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 0.75rem;
+}
+
+.reserveBtn {
+  border-style: none;
+  background-color: #0e72ed;
+  color: #fff;
+  padding: 0.5rem;
+  width: 20%;
+}
+
+.cancel-book {
+  border-style: none;
+  background-color: #fff;
+  color: #000;
+  padding: 0.5rem;
+  width: 20%;
```

Comment:
I suggest to keep styles for each component right beside that component. This was the recommended way of styling in Microverse lessons as far as I remember. What do you think? Any advantage of putting all styles in one file?